CMakeLists.txt

```diff
@@ -29,9 +29,11 @@ add_library(
   src/background/RoadGenerator.cpp
```

Comment:
Currently it is required to explicitly specify all directories with .cpp files that should be checked by clang tidy in the `add_tidy_target` function, please also add `src/Controllers/*`